zerver/lib/upload.py

```diff
@@ -381,18 +381,18 @@ def delete_message_image(self, path_id):
         logging.warning("%s does not exist. Its entry in the database will be removed." % (file_name,))
         return False
 
-    def upload_avatar_image(self, user_file, user_profile, email):
-        # type: (File, UserProfile, Text) -> None
-        email_hash = user_avatar_hash(email)
+    def upload_avatar_image(self, user_file, user_profile, uid):
+        # type: (File, UserProfile, int) -> None
+        uid_hash = user_avatar_hash(str(uid))
 
         image_data = user_file.read()
-        write_local_file('avatars', email_hash+'.original', image_data)
+        write_local_file('avatars', uid_hash+'.original', image_data)
 
         resized_data = resize_avatar(image_data)
-        write_local_file('avatars', email_hash+'.png', resized_data)
+        write_local_file('avatars', uid_hash+'.png', resized_data)
 
         resized_medium = resize_avatar(image_data, MEDIUM_AVATAR_SIZE)
-        write_local_file('avatars', email_hash+'-medium.png', resized_medium)
+        write_local_file('avatars', uid_hash+'-medium.png', resized_medium)
```

Comment:
stylistically we generally do a ` ` around `+` operators; might as well fix that while we're changing this.